### PR TITLE
APR-2930 Table structure for data enhancements

### PR DESCRIPTION
**Pull request by DevenKShah**

### CODE REVIEW

File: src/SFA.DAS.ApplyService.Database/Tables/OrganisationSectorExperts.sql [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	[SectorTrainingExperienceDuration] NVARCHAR(30) NULL,
+	[SectorTrainingExperienceDetails] NVARCHAR(MAX) NULL,
+	[IsQualifiedForSector] BIT NOT NULL,
+	[QualificationDetails] NVARCHAR(MAX) NULL,
+	[IsApprovedByAwardingBodies] BIT NOT NULL, 
+	[AwardingBodyNames] NVARCHAR(MAX) NULL, 
+	[HasSectorOrTradeBodyMembership] BIT NOT NULL, 
+	[SectorOrTradeBodyNames] NVARCHAR(MAX) NULL, 
+	[TypeOfApprenticeshipDelivered] NVARCHAR(50) NULL, 
+	[ExperienceInTrainingApprentices] NVARCHAR(100) NULL,
```
Comment (MarkFCain) [inline src/SFA.DAS.ApplyService.Database/Tables/OrganisationSectorExperts.sql:24]: ExperienceInTrainingApprentices needs to be 255 characters

**Review by MarkFCain — APPROVED** [review]

File: src/SFA.DAS.ApplyService.Database/Tables/OrganisationSectorExperts.sql (hunk 1 above)
Comment (DevenKShah) [reply src/SFA.DAS.ApplyService.Database/Tables/OrganisationSectorExperts.sql:24]: Following are the expected input for this, so I guess 100 should be sufficient 
image

File: src/SFA.DAS.ApplyService.Database/Tables/OrganisationAddresses.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿CREATE TABLE [dbo].[OrganisationAddresses]
+(
+	[Id] INT NOT NULL IDENTITY(1,1),
+	[OrganisationId] UNIQUEIDENTIFIER NOT NULL,
+	[AddressType] [TINYINT] NOT NULL,
+	[AddressLine1] NVARCHAR(MAX) NULL,
+	[AddressLine2] NVARCHAR(MAX) NULL,
+	[AddressLine3] NVARCHAR(MAX) NULL,
```
Comment (raviudari01) [inline src/SFA.DAS.ApplyService.Database/Tables/OrganisationAddresses.sql:8]: Do AddressLine1,2,3 needs to be Max size ?

**Review by raviudari01 — APPROVED** [review]

File: src/SFA.DAS.ApplyService.Database/Tables/OrganisationAddresses.sql (hunk 2 above)
Comment (DevenKShah) [reply src/SFA.DAS.ApplyService.Database/Tables/OrganisationAddresses.sql:8]: Address is not user input, we derive it from external API so it really does not matter.

**Pull request merged**